public/locales/en/panel-gettingstartedtour.json

```diff
@@ -0,0 +1,200 @@
+{
+  "previous-button": "Previous",
+  "next-button": "Next",
+  "finish-button": "Finish",
+  "mouse-descriptions": {
+    "altitude": "The altitude is displayed at the top left corner of the screen. Right click and drag to go closer or further away.",
+    "navigation": "The latitude and longitude are displayed at the top left corner of the screen. Left click and drag to orbit."
+  },
+  "components": {
+    "clear-sky-button": {
+      "info": "Is it hard to find? Try clearing the sky",
+      "label": "Clear Sky"
+    },
+    "current-altitude": {
+      "label": "Current altitude"
+    },
+    "current-focus": {
+      "label": "Current focus",
+      "none": "None"
+    },
+    "current-lat-long": {
+      "label": "Current position (lat, long)"
+    }
+  },
+  "tasks": {
+    "altitude": {
+      "label-higher": "Go to an altitude higher than {{altitude}} on {{object}}",
+      "label-lower": "Go to an altitude lower than {{altitude}} on {{object}}"
+    },
+    "delta-time": {
+      "label": "Change the delta time for the simulation speed"
+    },
+    "year": {
+      "label": "Change the year"
+    },
+    "focus": {
+      "label": "Focus on {{object}}"
+    },
+    "navigation": {
+      "label-lat": "Go to {{latitude}}° latitude",
+      "label-long": "Go to {{longitude}}° longitude",
+      "label-lat-long": "Go to {{latitude}}° latitude and {{longitude}}° longitude"
+    },
+    "pause-time": {
+      "label": "Pause the time!"
+    }
+  },
+  "steps": {
+    "introduction": {
+      "welcome": "Welcome to OpenSpace! Let's get started with exploring the observable Universe.",
+      "topics": {
+        "intro": "In this tutorial we will cover the following topics",
+        "sections": {
+          "navigation": "Navigation",
+          "time": "Time",
+          "content": "Content"
+        }
+      },
+      "about-tasks": {
+        "intro": "In each section, there will be some tasks designed to teach you how to use OpenSpace. Once you have fulfilled the task, the checkbox will be marked as completed.",
+        "task-label-example": "This is a task!",
+        "task-label-complete": "This is a completed task!"
+      },
+      "lets-go": "Let's dive right in!"
+    },
+    "navigation": {
+      "intro": {
+        "title": "Navigation",
+        "text": "In this chapter we will learn how to navigate in OpenSpace."
+      },
+      "zoom-in": {
+        "intro": "Let's go closer to Earth!"
+      },
+      "go-to-greenland": {
+        "intro": "Now let's go to Greenland! Rotate around the Earth until you find it."
+      },
+      "roll-camera": {
+        "intro": "You have two options for rolling the camera to change the orientation",
+        "option-scroll-wheel": "Scroll click and drag",
+        "option-shift": "Left click and drag + SHIFT"
```

Comment:
```suggestion
        "option-shift": "<keybind>shift</keybind> + Left click and drag"
```

?

Reply:
I was a bit reluctant to do this since there is already shift keybind component in the mouse hint thingy. Would be a bit "kaka på kaka" having it twice

![image](https://github.com/user-attachments/assets/41e64ebf-d983-403e-90e9-550058c705c2)

Reply:
It kind of explains what the illustration is showing. 

Maybe, rather remove it from the mouse hint thing?  I feel like this might need some redesign in general, though, so maybe leave it for now?

Reply:
Leaving it for now